submissions/5mountains/dom-api-task/js/main.js

```diff
@@ -0,0 +1,102 @@
+const styles = {
+    menuLink: 'menu__link',
+    menuLinkActive: 'menu__link--active',
+    menuItem: 'menu__item',
+    heroItem: 'hero__item',
+    heroItemActive: 'hero__item--active',
+    heroImg: 'hero__image',
+    itemIcon: 'item__icon',
+    itemTitle: 'item__caption',
+    itemPrice: 'item__price',
+    itemInfo: 'item__description'
+};
+const sectionMenu = document.querySelector('.nav__menu');
+const sectionHero = document.querySelector('.hero');
+const url = './data/data.json';
+let selectMenuItem;
+let selectHeroItem;
+
+async function fetchAsync () {
+    const response = await fetch(url);
+    const data = await response.json();
+    return data;
+}   
+  
+fetchAsync()
+    .then(data => {
+        const details = createHtml(data);
+        drawPage(details);
+        toggleTabs();
+    })
+    .catch(reason => console.log(reason.message));
```

Comment:
It's not a blocker but for displaying errors you can use `console.error`

Reply:
Changed it too!